here/here_readme.md

```diff
@@ -0,0 +1,218 @@
+- Determine the 85th percentile speed (to inform speed limit or enforcement initiatives).
+
+## Types of HERE data products
+HERE Technologies provides us with a few different datasets, such as:
+- traffic analytics tables containing probe data for all Toronto streets
+- traffic patterns tables containing aggregated speed information for all Toronto streets
+- street attribute tables showing speed limits, street names one way info (and more!) for Toronto's streets
+- many GIS layers of streets, intersections, points of interest, trails, rail corridors and water features.
+
+## HERE Data: dates and time periods
+As stated above, the traffic analytics table (here.ta) is updated daily. 
+
+Other products are generally updated quarterly, though this schedule has been somewhat disrupted due to the pandemic.
+
+There are also aggregated tables or views specific to certain modes (like trucks or cyclists) or time periods (like weekends or night-time).
+
+We have HERE data going back to 2012, but there sure weren't as many people walking around with "probes" in their pockets back then! Therefore, 2014 is generally seen as the first year for which travel times can be reasonably relied upon. 
+
+Travel times are calculated for time periods that are at least three weeks or longer. Shorter time periods will not generate enough accurate probe data. Longer time periods and roads with higher traffic volumes have more probe data, and therefore more accurate results, than short time periods on roads with light traffic. More probe data increases accuracy. There are some road segments in the City (cul-de-sacs and other lightly travelled roads) that have very few observations.
+
+# HERE data at a glance
+
+The Question       | The Answer     |
+:----------------- | :---------------
+What is the HERE dataset used for? | To calculate travel times and monitor congestion (mostly)
+Where is the HERE dataset from? | HERE Technologies, via an agreement with the Ontario Ministry of Transportation (MTO)
+Is it available on Open Data? | No
+What area does the HERE dataset represent? | All of Toronto; the co-ordinate system is EPSG: 4326
+Where is it stored? | On an internal postgres database called bigdata; in several schema (here, here_analysis, here_eval and here_gis)
+Are there any naming conventions? | If you see `_##_#` at the end of a table name (like `streets_21_1`) the first number is the year, and the second number is the quarter.
+How often is it updated? | Probe data are updated every day; reference files are usually updated quarterly
+How long are the time bins? | 5 minutes
+How far back does it go? | To 2012, but 2014 data are much more accurate
+What are the limitations? | Travel times are generally calculated for time periods lasting three or more weeks; use data from 2014 onward
+What can I get? | Custom aggregations
+Are raw data available? | No
+How do I cite HERE data? | ???
```

Comment:
Is this supposed to be for team members or... the general public?

Reply:
Delete the citing reference
Split "what can I get" into working / not working for the City